package.json

```diff
@@ -6,7 +6,9 @@
   "dependencies": {
     "@readme/variable": "^7.2.1",
     "codemirror": "^5.48.2",
+    "prop-types": "^15.7.2",
     "react": "^16.13.1",
+    "react-codemirror2": "^7.2.1",
```

Comment:
Where's this dep being used?

Reply:
Right [here](https://github.com/readmeio/syntax-highlighter/blob/feat/codeeditor-export/src/codeEditor/index.jsx#L3)!